src/v2/BaseLoginRouter.js

```diff
@@ -133,20 +137,17 @@ export default Router.extend({
           ]
         };
       }
-      // Need to mimic IdxResponses as idx returns raw response at error case
-      return this.handleIdxResponseSuccess({
-        rawIdxState: error.details,
-        context: _.pick(error.details, 'messages'),
-        neededToProceed: [],
-      });
+      // Pull all nested messages to the root context
```

Comment:
why needs to do this? (responseTransformer shall have done it?)

Reply:
This is needed for the errors we create by hand - specifically the `oie.feature.disabled`, and `oform.error.unexpected` generic errors.

As an alternative, I can populate the `context` above during error creation - if we think that's more straightforward.

Reply:
ah, I see. yeah, seems more straightforward to set `context` (strictly mimic what idx-js response is)